### PR TITLE
docs: update README

### PR DESCRIPTION
**Pull request by kenjis**
**Description**
- minor updates

**Checklist:**
- [x] Securely signed commits
- [] Component(s) with PHPDoc blocks, only if necessary or adds value
- [] Unit testing, with >80% coverage
- [] User guide updated
- [] Conforms to style guide

### CODE REVIEW

File: admin/framework/README.md [hunk 1]
```diff
@@ -11,8 +11,8 @@ It has been built from the
 
 More information about the plans for version 4 can be found in [CodeIgniter 4](https://forum.codeigniter.com/forumdisplay.php?fid=28) on the forums.
 
-The user guide corresponding to the latest version of the framework can be found
-[here](https://codeigniter4.github.io/userguide/).
+You can read the [user guide](https://codeigniter4.github.io/userguide/)
```
Comment (datamweb) [inline admin/framework/README.md:14]:
```suggestion
You can read the [user guide](https://codeigniter.com/user_guide/)
```

File: admin/starter/README.md [hunk 2]
```diff
@@ -11,8 +11,8 @@ It has been built from the
 
 More information about the plans for version 4 can be found in [CodeIgniter 4](https://forum.codeigniter.com/forumdisplay.php?fid=28) on the forums.
 
-The user guide corresponding to the latest version of the framework can be found
-[here](https://codeigniter4.github.io/userguide/).
+You can read the [user guide](https://codeigniter4.github.io/userguide/)
```
Comment (datamweb) [inline admin/starter/README.md:14]:
```suggestion
You can read the [user guide](https://codeigniter.com/user_guide/)
```

**Review by datamweb — COMMENTED** [review]
This is a personal preference. I prefer to see the documents that have been Deployed through the **codeigniter.com**

File: admin/framework/README.md (hunk 1 above)
Comment (kenjis) [reply admin/framework/README.md]: Done.

File: admin/starter/README.md (hunk 2 above)
Comment (kenjis) [reply admin/starter/README.md]: Done.

**Review by datamweb — APPROVED** [review]
LGTM!

**Review by paulbalandan — APPROVED** [review]

**Pull request merged**